vid/src/advz/test.rs

```diff
@@ -254,51 +263,9 @@ fn verify_share_with_multiplicity() {
     let (shares, common, commit) = (disperse.shares, disperse.common, disperse.commit);
 
     for share in shares {
-        assert!(advz.verify_share(&share, &common, &commit).unwrap().is_ok())
-    }
-}
-
-#[test]
-fn sad_path_verify_share_with_multiplicity() {
-    // regression test for https://github.com/EspressoSystems/jellyfish/issues/654
-    let advz_params = AdvzParams {
-        recovery_threshold: 16,
-        num_storage_nodes: 20,
-        max_multiplicity: 32, // payload fitting into a single polynomial
-        payload_len: 8200,
-    };
-    let (mut advz, payload) = advz_init_with::<Bn254>(advz_params);
-
-    let disperse = advz.disperse(payload).unwrap();
-    let (shares, common, commit) = (disperse.shares, disperse.common, disperse.commit);
-    for (i, share) in shares.iter().enumerate() {
-        // corrupt the last evaluation of the share
```

Comment:
Do we maybe want to keep some version of this test? Seems important to exercise the sad path for `multiplicity > 1`

Reply:
done in https://github.com/EspressoSystems/jellyfish/pull/678/commits/52e7c17e87bf4b7c88ef0703163684dfb054fa14. All tests now use nontrivial multiplicity. (Set in `advz_init`.)